src/stratis_cli/_actions/_utils.py

```diff
@@ -92,3 +104,47 @@ def fetch_property(props, name):
         return variant
     except KeyError as err:  # pragma: no cover
         raise StratisCliPropertyNotFoundError(name) from err
+
+
+def get_clevis_info(namespace):
+    """
+    Get clevis info, if any, from the namespace.
+
+    :param namespace: namespace set up by the parser
+
+    :returns: clevis info or None
+    :rtype: pair of str or NoneType
+    """
+    clevis_info = None
+    if namespace.clevis is not None:
+        if namespace.clevis in ("nbde", "tang"):
+            if namespace.tang_url is None:
+                raise StratisCliMissingClevisTangURLError()
+
+            clevis_config = {CLEVIS_KEY_URL: namespace.tang_url}
+            if not namespace.trust_url and namespace.thumbprint is None:
+                raise StratisCliMissingClevisThumbprintError()
+
+            if (
+                # pylint: disable=bad-continuation
+                namespace.trust_url
+                and not namespace.thumbprint is None
+            ):  # pragma: no cover
+                raise AssertionError("--trust-url and --thumbprint both set")
```

Comment:
Is this necessary given that these arguments are specified as mutually exclusive in the parser?

Reply:
It's trying to hold the spot that an ```unreachable``` would in Rust. I've tidied it up in the next commit.